### PR TITLE
Add coloring feature to nodes

### PR DESCRIPTION
**Pull request by Greybird**
The purpose of this PR is to allow configuring colors for nodes (either leaves or intermediate "directories") based on regular expressions.

# Screenshots
## Configuration
image

## Results
image

### CODE REVIEW

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-colors.71.build.712 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/35532445) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/a8d06c0b75 by @)

**Comment by ekaliroots** [comment]
Nice feature, we may add style like bold on populated queues too

**Review by SeanFeldman — DISMISSED** [review]
Thank you, @Greybird !

File: src/ServiceBusExplorer/Forms/MainForm.cs [hunk 1]
```diff
@@ -5772,6 +5791,38 @@ private TreeNode FindNode(string path, TreeNode node)
             return null;
         }
 
+        void ReApplyColors(TreeNode parentNode)
```
Comment (ErikMogensen) [inline src/ServiceBusExplorer/Forms/MainForm.cs:5794]: A picky comment : Reapply instead of ReApply since reapply is a word in itself.

**Review by ErikMogensen — CHANGES_REQUESTED** [review]
Awesome feature!

Just that minor casing issue.

File: src/ServiceBusExplorer/Forms/MainForm.cs (hunk 1 above)
Comment (Greybird) [reply src/ServiceBusExplorer/Forms/MainForm.cs]: Thanks for pointing this, I learned something 👍 !

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-colors.72.build.713 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/35539755) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/d3be126f00 by @)

**Review by ErikMogensen — DISMISSED** [review]

**Comment by ErikMogensen** [comment]
@Greybird, I merged the other PR, which you said could adapt. I hope you can fix this PR instead.

@SeanFeldman, apparently you have to approve again when there has been changes to the PR. I think that is new behavior.

**Comment by SeanFeldman** [comment]
@ErikMogensen, no worries. I'll let @Greybird address this PR and then re-approve it so that there's no need to do it a 3rd time 😀

**Comment by ErikMogensen** [comment]
Sounds good @SeanFeldman 😃

**Comment by SeanFeldman** [comment]
:white_check_mark: [Build ServiceBusExplorer 4.2.0-colors.74.build.717 completed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/35603247) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/de7d1f15ff by @)

**Comment by Greybird** [comment]
> @Greybird, I merged the other PR, which you said could adapt. I hope you can fix this PR instead.
> 
@ErikMogensen, of course, no issue! Thanks for the quick feedback and merge of the other PR!

**Review by ErikMogensen — APPROVED** [review]
Thanks @Greybird !

**Review by SeanFeldman — APPROVED** [review]

**Comment by paolosalvatori** [comment]
Sorry guys for coming late, I was terribly busy during this period. Thanks @SeanFeldman @ErikMogensen

**Pull request merged**